packages/app/src/file-system/requests/get-paths.request.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { ipcRenderer } from "electron-better-ipc"
+import { ResultObject } from "App/core/builder"
+import { IpcDialogFileSystemRequest } from "App/file-system/constants"
+import { GetPathInput } from "App/file-system/dto"
+
+export const getPathRequest = async (
```

Comment:
Seems to me that this should be `getPathsRequest`

Reply:
https://github.com/mudita/mudita-center/pull/907/commits/9c0bda2b2d75f8e396e1c22a11ac4efc41b4ef67